packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -13,14 +13,18 @@
 import {AriaButtonProps} from '@react-types/button';
 import {HTMLAttributes, RefObject} from 'react';
 import {MenuTriggerState} from '@react-stately/menu';
-import {useId} from '@react-aria/utils';
+import {MenuTriggerType} from '@react-types/menu';
+import {mergeProps, useId} from '@react-aria/utils';
+import {useLongPress} from '@react-aria/interactions';
 import {useOverlayTrigger} from '@react-aria/overlays';
 
 interface MenuTriggerAriaProps {
   /** The type of menu that the menu trigger opens. */
   type?: 'menu' | 'listbox',
   /** Whether menu trigger is disabled. */
-  isDisabled?: boolean
+  isDisabled?: boolean,
+  /** How menu is triggered. */
+  trigger?: MenuTriggerType
```

Comment:
A bit surprised that `MenuTriggerAriaProps` doesn't extend MenuTriggerProps or anything... Open question for the team: should we refactor the props for MenuTrigger?  Would help sanitize `useMenuTriggerState` props in the process.